core/src/main/java/com/amazon/opendistroforelasticsearch/sql/expression/operator/predicate/BinaryPredicateOperator.java

```diff
@@ -210,8 +260,85 @@ private static FunctionResolver equal() {
     );
   }
 
-  private static Map<FunctionSignature, FunctionBuilder> predicateFunction(
+  private static FunctionResolver notEqual() {
```

Comment:
could we define notEqual as the combination of not(equal)

Reply:
I was also thinking about it didn't figure it out yet